lib/game.rb

```diff
@@ -0,0 +1,71 @@
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 4, 8],
+    [6, 4, 2],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8]
+  ]
+
+  def initialize(player_1=Human.new("X"), player_2=Human.new("O"), board=Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    self.board.turn_count % 2 == 0 ? self.player_1 : self.player_2
+  end
+
+  def over?
+    true if won? || draw?
```

Comment:
Avoid returning boolean literals from `if` statements, relying instead on the implicit return value of the boolean expression.
```
def over?
  won? || draw?
end
```